src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,18 @@
+// /src/redux/categories/categoriesSlice.js
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkstatus: () => 'Under construction',
+  },
+});
+
+export const checkstatus = (state) => state.categories.status;
```

Comment:
- Good job done so far, but according to the project requirements,  this function should be an action inside your reducer, do not forget to export it also. Consider doing something like this:
```JavaScript
const categoriesSlice = createSlice({
  name: 'categories',
  initialState,
  reducers: {
    checkstatus: (state) => state.status,
  },
});
```